packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -189,6 +190,11 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     let hasCheckmarks = !!(items && items.some(item => !!item.canCheck));
     const submenuProps = this.state.expandedMenuItemKey ? this._getSubmenuProps() : null;
 
+    let contextMenuWidth;
+    if (useTargetWidth && this._target as HTMLElement) {
```

Comment:
What if _target is not an `HTMLElement` but a `MouseEvent` can this happen or is the typing wrong?